src/components/Hero/HeroSection.astro

```diff
@@ -0,0 +1,107 @@
+---
+import CountUp from "react-countup";
+import GridWrapper from "./GridWrapper";
+import HeroSwiper from "./HeroSwiper";
+---
+
+<>
+  {/* Text of the HeroSection */}
+  <div class="flex max-md:grid">
+    <div class="p-10 w-[50%] max-md:w-[100%]">
```

Comment:
we are doing mobile first. and u are doing it in reverse
it should be grid md:flex 
not the other way around